### PR TITLE
feat: Add audio only mode to 'More Settings'

### PR DESCRIPTION
**Pull request by Ragzzy-R**

### CODE REVIEW

File: src/components/MoreSettings/MoreSettings.jsx [hunk 1]
```diff
@@ -89,6 +93,12 @@ export const MoreSettings = () => {
             </Dropdown.Item>
           )}
           <FullScreenItem hoverStyles={hoverStyles} />
+          <Dropdown.Item onClick={() => toggleIsAudioOnly()} css={hoverStyles}>
```
Comment (raviteja83) [inline src/components/MoreSettings/MoreSettings.jsx:96]: move this to a separate component and use useContext there.

File: src/components/VideoTile.jsx [hunk 2]
```diff
@@ -52,14 +55,14 @@ const Tile = ({ peerId, showStatsOnTiles, width, height }) => {
             />
           ) : null}
 
-          {track ? (
+          {track && !isAudioOnly ? (
```
Comment (raviteja83) [inline src/components/VideoTile.jsx:58]: this might not turn off the incoming data. check in `chrome://webrtc-internals` to verify if that is the. case

Comment (triptu) [reply src/components/VideoTile.jsx:58]: it should, if the video was on the unmount useeffect will detach and if it's not on already, attach will never be called

Comment (Ragzzy-R) [reply src/components/VideoTile.jsx:58]: me and @raviteja83  checked webrtc-internal and we were still getting packets when video was unmounted, so we are now passing an extra `attach` prop to `Video`. Please check [this PR](https://github.com/100mslive/web-sdks/pull/323)

I've raised a PR against websdk repo. Closing this PR.

**Comment by Ragzzy-R** [comment]
mistakenly raised a PR against this Repo. Raised [PR](https://github.com/100mslive/web-sdks/pull/323) against webSDK now. Closing this PR.

**Pull request closed without merging**